metrics/src/vespa/metrics/metric.h

```diff
@@ -87,7 +87,7 @@ struct Tag
 
     Tag(vespalib::stringref k);
     Tag(vespalib::stringref k, vespalib::stringref v);
-    Tag(const Tag &);
+    Tag(const Tag &) noexcept;
     Tag & operator = (const Tag &);
     Tag(Tag &&) = default;
```

Comment:
noexcept on the move constructors too ?

Reply:
PR #23328 adds noexcept specifier for the move constructor.